src/web/routes/application/check/get.test.js

```diff
@@ -68,7 +80,11 @@ test('getFlattenedRowData returns flattened row data', (t) => {
   const result = getFlattenedRowData(req)(steps)
 
   t.deepEqual(result,
-    [{ keyA: 'myKey1', valueA: 'myValue1', path: 'mypath1' }, { keyB: 'myKey3', valueB: 'myValue3', path: 'mypath1' }, { key2: 'myKey2', value: 'myValue2', path: 'mypath2' }],
+    [{ keyA: 'myKey1', valueA: 'myValue1', path: 'mypath1' }, { keyB: 'myKey3', valueB: 'myValue3', path: 'mypath1' }, {
```

Comment:
Is there a reason for this reformatting?

Reply:
It was done by webstorm, I'll put it back

Reply:
Webstorm reformatted because it's over 160 lines, I've put it back but we might want to add a max line length?